Examples/Tests/ParticleDataPython/PICMI_inputs_2d.py

```diff
@@ -1,6 +1,8 @@
+#!/usr/bin/env python3
+
 from pywarpx import picmi
-import numpy as np
```

Comment:
Can you undo this change? This will cause a conflict with PR #2637. That PR reorders the imports to the standard style, with the system packages first.

Reply:
Sure, no prob! :) Fixed!